.travis.yml

```diff
@@ -4,6 +4,9 @@ node_js:
   - '6'
   - '4'
 before_install: if [[ `npm -v` < 3* ]]; then npm i -g npm@^3; fi
+script:
+  - npm run lint
```

Comment:
I don't like this TBH. Especially the fact it's now duplicated in 2 yml files.

Let's do everything we need in `package.json` with `pre` and `post` scripts.

Reply:
https://github.com/okonet/lint-staged/pull/250#pullrequestreview-60625243

Reply:
Yeah, so the @sudo-suhas was asking to add `test:watch` and I totally understand why we need it but I don't think this is an improvement to what we had. Sorry I didn't make it clear in the review but I'm trying to spend less time on OSS ;)